### DISCUSSION TITLE
Output issues

### DISCUSSION THREAD

**Discussion by Bhavak29** [adilsoybali/Log4j-RCE-Scanner#11] [Q&A] [answered]
Hi devs,
I recently started to use your tool. I was testing for log4j on a target and got a DNS request from domain.burpcollaborator.com in m collaborator client that too after an hour. However, when I tried to implement all three ways i.e. using API and user-agent headers and test parameter manually, it didn't give me any DNS ping in my collaborator. Can you please explain to me where I went wrong?

**Comment by adilsoybali** [comment, marked as answer]
Do you get the same output when you scan again with this tool in the same scope?

**Reply by Bhavak29** [reply]
Yes, I get the same domain.burpcollaborator.com.

**Reply by adilsoybali** [reply]
You are probably sending the http request wrong. This is not a problem with the tool. I can help if you contact me with screenshots or video if possible from discord.
